api/lib/opentelemetry/trace/tracer_factory.rb

```diff
@@ -8,6 +8,10 @@ module OpenTelemetry
   module Trace
     # No-op implementation of a tracer factory.
     class TracerFactory
+      HTTP_TEXT_FORMAT = DistributedContext::Propagation::HTTPTextFormat.new
```

Comment:
I see that https://github.com/open-telemetry/opentelemetry-specification/pull/313 moved span_processors. Does that necessarily mean text format and binary format need to move here as well?

Reply:
> All configuration objects (SDK specific) and extension points (span processors,
**propagators**) must be provided to the `TracerFactory`.

The text mentions moving propagators as well.